modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/FormRenderer/FormRenderer.es.js

```diff
@@ -60,6 +60,16 @@ class FormRenderer extends Component {
 	_handleFieldFocused(event) {
 		this.emit('fieldFocused', event);
 	}
+
+	_getDisplayableFn() {
```

Comment:
Hi @linolaoj. The PR looks good to me. Just this method must be alphabetically ordered. =) Thank you!